src/index.html

```diff
@@ -7,8 +7,36 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+
+      <div class="logo"><a href="#"><img src="images/logo.png" alt="logo img"></a></div>
```

Comment:
Do you need div here?
Use only semantic tags
```suggestion
      <a href="#">
        <img src="images/logo.png" alt="Moyo logo">
      </a>
```